R/get-pit-histogram.R

```diff
@@ -37,7 +36,7 @@
 #' Rosalind M. Eggo, W. John Edmunds (2019) Assessing the performance of
 #' real-time epidemic forecasts: A case study of Ebola in the Western Area
 #' region of Sierra Leone, 2014-15, \doi{10.1371/journal.pcbi.1006785}
-get_pit_histogram <- function(forecast, num_bins = "auto", breaks = NULL, by,
+get_pit_histogram <- function(forecast, num_bins, breaks = NULL, by,
```

Comment:
we can probably do away with the default argument for `breaks`, too, here.

Reply:
My personal logic is something like: 

if there is a `NULL` default then that tells me, as a user, that I don't have to provide any argument there. If there is no default, I think "I need to pass an arg here". So my personal preference would be for `num_bins` to have a default as well - it just doesn't work because we have two different defaults for the methods. 

But then again I think you're right - we can just keep the default for the methods and not have it for the generic.